tests/unit/helpers.py

```diff
@@ -6,9 +6,34 @@
 import requests
 import responses
 
+from gitlab import base
+
 MATCH_EMPTY_QUERY_PARAMS = [responses.matchers.query_param_matcher({})]
 
 
+class FakeObject(base.RESTObject):
+    pass
+
+
+class OtherFakeObject(FakeObject):
+    _id_attr = "foo"
+
+
+class FakeManager(base.RESTManager):
+    _path = "/tests"
+    _obj_cls = FakeObject
```

Comment:
There is still some duplication between this and the fake classes in the mixin tests, as well as in individual test cases, but maybe best to do that cleanup later so the diff doesn't get too crazy.